install/v2plugin/config.template

```diff
@@ -103,6 +103,14 @@
           ],
           ## Do not change the default value, this will be replaced with $CONTIV_V2PLUGIN_NAME
           "Value": "__CONTIV_V2PLUGIN_NAME__"
+       },
+       {
+          "Description": "Forwarding mode for netplugin",
+          "Name": "fwd_mode",
```

Comment:
update https://github.com/contiv/netplugin/tree/master/install/v2plugin

Reply:
check if you need to configure Fabric mode(ACI/default) also here

Reply:
it's fine for now about fabric mode